web-registry/src/pages/Roles/hooks/useRolesSubmit.tsx

```diff
@@ -0,0 +1,209 @@
+import { useCallback } from 'react';
```

Comment:
another important file to review
when submitting roles form, we store the project developer info both:
- in the offchain db (through `wallet` and `party` tables, `project` table having some `developer_id` foreign key to the `party` table, hence the logic with update/create wallet/party)
- in the anchored project metadata (ie the metadata with the iri referenced on chain)